.editorconfig

```diff
@@ -0,0 +1,16 @@
+# http://editorconfig.org
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.py]
+indent_size = 4
+
+[*.md]
+trim_trailing_whitespace = false
+insert_final_newline = false
```

Comment:
Why are these both false for markdown? Why not make them true (or more concisely, simply drop the markdown specific section since it is identical to the global config)

Reply:
These are only common writing formats for some special files. I think it is fine not to add blank line or remove any whitespace characters preceding newline characters in Markdown file. You can set what you want. Just keep all the files consistent. Thanks.